client/snapctl.go

```diff
@@ -41,7 +55,14 @@ type snapctlOutput struct {
 }
 
 // RunSnapctl requests a snapctl run for the given options.
-func (client *Client) RunSnapctl(options *SnapCtlOptions) (stdout, stderr []byte, err error) {
+func (client *Client) RunSnapctl(stdin io.Reader, options *SnapCtlOptions) (stdout, stderr []byte, err error) {
```

Comment:
This should be `RunSnapctl(options *SnapCtlOptions, stdin io.Reader) ` and if stdin is not needed we use "nil".